boards/mcb2388/Makefile.include

```diff
@@ -0,0 +1,10 @@
+USEMODULE += boards_common_msba2
+
+# FLASHER will be lpc2k_pgm still
+DEBUG_ADAPTER ?= dap
+PROGRAMMER ?= openocd
+
+JLINK_DEVICE := LPC2388
```

Comment:
I don't think `:=` is needed, just use a simple `=`.

Reply:
[With `:=` the variables are expended one, with `=` they are expended on each use.](https://www.gnu.org/software/make/manual/html_node/Flavors.html#Flavors) I read this as `:=` is faster, if it can be used.

Reply:
Yea, it's not needed here, just a habit I developed after I had variables change unintentionally when some other variables they were composed of changed. So I prefer `:=` as it's more predictable.

But in this case it doesn't matter.